WaypointMSG[] waypoints
---
bool success
string msg
